cmd/security-secretstore-setup/secretstore-init.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/dumb-init /bin/sh
+set -e
+
+
+# this is the placeholder for secretstore-init based on the discussion with Bryon ( see details in issue #1673 of edgex-go)
+# the name and content may be changed later
```

Comment:
just in case we care, no newline at the end

Reply:
BUG!  Add

```sh
exec ./security-secretstore-setup --init=true
```

to the end of this line.  Otherwise the container will just run this script script and do nothing!

Reply:
I may misunderstand you @bnevis-i since I thought we need to call the script in which all the binaries will be called. Currently secretstore-init.sh will have nothing inside. When we implement security-sercretstore-setup we will just add the binary into the file.

Reply:
See discussion for line 1 of this file above.

Reply:
As I mentioned above this is not a bug but a holder project. This line needs to be added into different place depending on the result of future discussion.

Reply:
Changed back to original method of calling binary.